frontend/src/app/features/calendar/te-calendar/te-calendar.component.sass

```diff
@@ -102,6 +99,12 @@ te-calendar
         .fc-fadeout
           display: none
 
+    .te-calendar--add-entry,
+    .te-calendar--time-entry
+      .fc-event-title-container
+        margin: 0 !important
+        line-height: 14px !important
```

Comment:
if we're already using important I think we can remove the extra selector up top

Reply:
I'd rather leave it in as we can't be sure that the `fc-event-title-container` is not used elsewhere since it is coming from FullCalendar and we all know that their DOM structure is not always logical 🤷‍♀️